doc/cookbook/source/examples/classifier/mkl_binary_class.rst

```diff
@@ -0,0 +1,59 @@
+==================================================
+Multiple Kernel Learning for Binary Classification
+==================================================
+
+Multiple kernel learning (MKL) is about using a combined kernel i.e. a kernel consisting of a linear combination of arbitrary kernels over different domains.
+
+.. math::
+
+    f({\bf x})=\text{sign} \left(\sum_{i=0}^{N-1} \alpha_i k({\bf x}, {\bf x_i})+b\right)
+
+where :math:`{\bf x_i},{i = 1,...,N}` are labeled training examples where label is :math:`y_i \in {±1}`, :math:`\bf x` is the new data point, :math:`k` is a kernel function and :math:`b` is bias.
+
+One could make a combination of kernels like:
+
+.. math::
+
+    {\bf k}(x_i,x_j)=\sum_{k=0}^{K} \beta_k {\bf k_k}(x_i, x_j)
+
+where :math:`\beta_k > 0` and   :math:`\sum_{k=0}^{K} \beta_k = 1`
+
+Optimal :math:`\alpha_i`, :math:`\beta` and :math:`b` are determined via training.
```

Comment:
I think this cookbook should solely focus on the MKL part
 * settup up multiple kernels
 * running mkl
 * extracting kernel weights

And refer to the kernel svm cookbook for all other things